RFS/test-resources/inventory.md

```diff
@@ -13,3 +13,107 @@ An Elasticsearch 7.10 snapshot repo containing a single snapshot (`global_state_
 #### ES_7_10_Double
 An Elasticsearch 7.10 snapshot repo containing a two snapshots (`global_state_snapshot`, `global_state_snapshot_2`).  `global_state_snapshot` contains two indices (`posts_2023_02_25`, `posts_2024_01_01`), each with few documents in them, an index template, `posts_index_template`, and a composite template, `posts_template`.  `global_state_snapshot_2` is the same, except it doesn't have the index `posts_2023_02_25` in it.
 
+#### ES_7_10_Updates_Deletes_w_Soft
```

Comment:
SnapshotStateTest allow for this kind of configuration, lets update those tests rather than check in artifacts

Reply:
I think I'm gonna disagree and commit on this one.  The fewer containers we need to spin up in order to run our tests, the better.  When we use these instead of spinning up a Source cluster it runs in seconds instead of minutes.

Reply:
I'll make a JIRA for the test-resource generation idea; that makes a lot of sense!